roles/servicetelemetry/templates/manifest_grafana_v5.j2

```diff
@@ -0,0 +1,95 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: {{ ansible_operator_meta.name }}-grafana
+  namespace: {{ ansible_operator_meta.namespace }}
+  labels:
+    dashboards: "stf"
+spec:
+  serviceAccount:
+    metadata:
+      annotations:
+        serviceaccounts.openshift.io/oauth-redirectreference.primary: '{{ grafana_oauth_redir_ref | to_json }}'
+{% if servicetelemetry_vars.graphing.grafana.ingress_enabled is defined and servicetelemetry_vars.graphing.grafana.ingress_enabled %}
```

Comment:
No more ingressEnabled parameter that makes this simpler :) With Grafana Operator v5 there is more of an override approach than providing a bunch of custom parameters in the CRD. Overall I like this approach better.